src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -118,12 +120,16 @@ function ExportAnnotations({
     e.preventDefault();
 
     try {
+      const format = exportFormat.value;
       const annotationsToExport =
         selectedUser?.annotations ?? exportableAnnotations;
-      const filename = `${customFilename ?? defaultFilename}.json`;
-      const exportData =
-        annotationsExporter.buildExportContent(annotationsToExport);
-      downloadJSONFile(exportData, filename);
+      const filename = `${customFilename ?? defaultFilename}.${format}`;
+
+      if (format === 'json') {
```

Comment:
Only handling JSON format for now. The rest of formats are disabled, and JSON is the default.

Reply:
Showing the item but disabling it suggests to the user there is some condition under the user's control which would make it available. If we decide to ship with a smaller set of formats initially, I would simply remove the unimplemented options from the menu.

Since this is behind a feature flag, we could present the options as disabled, but note that they aren't ready yet. I had a quick try at this, but found an issue that we need to make the listbox align to the right edge of the button rather than the left, in case it needs to be larger than the button:

<img width="427" alt="Not yet implemented" src="https://github.com/hypothesis/client/assets/2458/5429c11d-ccf5-4250-83e9-05462d39a587">